src/Comparer/StubValueComparer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Diff\Comparer;
+
+/**
+ * @since 3.0
+ *
+ * @license GPL-2.0+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class StubValueComparer implements ValueComparer {
```

Comment:
Maybe it was a mistake to put the file in the "src" folder then, you have to tell me.

I don't see why it should not be possible to write meaningful test cases with the StrictComparer. I can do this in a later patch after this was merged, if you don't mind me uploading a pull request that deletes the stub class.